.gitignore

```diff
@@ -42,3 +42,21 @@ hyperspy/io_plugins/unbcf_fast.c
 hyperspy/misc/etc/test_compilers.obj
 /.cache/
 /.spyproject/
+test/test background.ipynb
```

Comment:
A bunch of the files in your gitignore are being merged. You should remove .gitignore from the PR

Reply:
Oh yes thanks. 
And thank you for all your comments !

Reply:
Honestly I don't know how to delete undesirable files as .gitignore. 

Should I remove those files in my local repository and start a new PR ? 

Sorry for this !